idl/bulk_load.thrift

```diff
@@ -91,7 +91,7 @@ struct bulk_load_request
 {
     1:dsn.gpid                pid;
     2:string                  app_name;
-    3:dsn.rpc_address         primary_addr;
+    3:dsn.rpc_address         primary;
```

Comment:
why change this name？could you provide some context message for it？

Reply:
In some newly added macros, they suppose the rpc_address and host_port fields are naming similarly with name `xxx` and `hp_xxx`. The macros are helpful to ensure the operations are taken both on the 2 fields. So I rename it.

Rename the thrift structure fileds dosen't break compatibility if not change the sequence number.